### DISCUSSION TITLE
Interactive chart edits

### DISCUSSION THREAD

**Discussion by Hrishikesh007788** [sinaptik-ai/pandas-ai#690] [Q&A]
@gventuri  Can we get a demo on how this works?

**Comment by gventuri** [comment]
@Hrishikesh007788 basically it just leverages the `Agent.chat()` method. It keeps in memory the previous messages and is then able to apply changes.
This could be particularly useful, for example, in a streamlit implementation.

Only make sure you use the same instance of an agent. If you have a custom frontend, you might implementa a way to save a memory and load it back in the agent as needed.
